libraries/AP_Follow/AP_Follow.h

```diff
@@ -194,14 +194,15 @@ class AP_Follow
     //==========================================================================
 
     AP_Int8     _enabled;           // 1 = Follow mode is enabled; 0 = disabled
-    AP_Int16    _sysid;             // MAVLink system ID of the target (0 = auto-select first sender)
+    AP_UInt32   _sysid;             // MAVLink system ID of the target (0 = auto-select first sender)
```

Comment:
Why the change in type? It would need param conversion too.

Reply:
Requested by @tridge - he wants to do this to start moving to SYSID being IP addresses, he requested this the last time it came up in the in the devcall.

Reply:
Ok simplified it to just return the sysid cast as a uint32_t.